src/Components/Nav/ProductDetail/ContentLeft/ReviewList/ReviewList.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faChevronLeft,
+  faChevronRight,
+} from "@fortawesome/free-solid-svg-icons";
+import "./ReviewList.scss";
+import DetailList from "./Components/DetailList";
+
+class ReviewList extends Component {
+  constructor(props) {
+    super(props);
+    this.Slide = React.createRef();
+  }
+
```

Comment:
```suggestion
Slide = React.createRef();
```
createRef 도 constructor 안에서 만들어주실 필요 없습니다~ 바벨이 잘 처리해줍니다

Reply:
넵! 수정하겠습니다